packages/babel-plugin-transform-optional-chaining/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "babel-plugin-transform-optional-chaining",
+  "version": "7.0.0-alpha.13",
+  "description": "Transform optional chaining operators into a series of nil checks",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-optional-chaining",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "dependencies": {
+    "babel-plugin-syntax-optional-chaining": "7.0.0-alpha.13"
+  },
+  "keywords": [
+    "babel-plugin"
```

Comment:
@xtuc [It's not the Elvis operator](https://github.com/babel/babel/issues/855#issuecomment-75444738).